YANG/ccamp/Client-signal-yang/ietf-eth-tran-types.yang

```diff
@@ -314,32 +315,53 @@ module ietf-eth-tran-types {
         "Coupling Flag.";
     }
   }
-  
+
   identity topology-role {
-    description 
-    "The rold of underlay topology, e.g., hub, spoke, any-to-any. ";
+    description
+    "The rold of underlay topology, e.g.,
+    hub, spoke, any-to-any. ";
   }
-  
+
   identity resilience {
-    description 
-    "Placeholder for resilience information in data plane, for future study. ";
+    description
+    "Placeholder for resilience information in data plane,
+    for future study. ";
   }
-  
+
   identity access-role {
-    description 
+    description
     "Indicating whether the access is a working or protection access.";
   }
-  
+
+  identity root-primary {
+    base access-role;
+    description
+      "Designates the primary root UNI of an E-Tree service, and may also
+	   designates the UNI access role of E-LINE and E-LAN service.";
+  }
+
+  identity root-backup {
+    base access-role;
+    description
+      "Designates the backup root UNI of an E-Tree service.";
+  }
+
+  identity leaf-access {
+    base access-role;
+    description
+      "Designates the leaf UNI of an E-Tree service.";
+  }
```

Comment:
Based on the discussion above, the root versus leaf role should be defined on a service end-point basis either through explicit configuration or by proper configuration of the split-horizon group.

The access-role identityrefs should only define the role of the service access point within the access protection configuration (primary or secondary)

Reply:
```suggestion
  identity primary {
    base access-role;
    description
      "Designates the primary access link (UNI) of an Ethernet
      service end-point.";
  }

  identity secondary {
    base access-role;
    description
      "Designates the secondary access link (UNI) of an Ethernet
      service end-point.";
  }
```